Packs/GitLab/Playbooks/Gitlab_-_Guest_user_permission_change.yml

```diff
@@ -512,7 +512,7 @@ tasks:
       RuleName:
         simple: XSOAR - Block IP playbook - ${alert.alertid}
       UserVerification:
-        simple: "False"
+        simple: ${inputs.UserVerification}
```

Comment:
Can you confirm if this was intended?
In some we have hardcoded values and in others a playbook input.